README.md

```diff
@@ -1,2 +1,26 @@
 # LinQuadOptInterface.jl
-Intermediate wrapper for MOI of CPLEX, Gurobi and Xpress
+
+LinQuadOptInterface is a intermediate wrapper designed to bridge low-level Integer Linear and Quadratic solvers. I provides access to many MathOptInterface functionalities mainly related to problem modifications: 
```

Comment:
I provides -> It provides.
Maybe also "LinQuadOptInterface" -> "LinQuadOptInterface (LQOI)"